### DISCUSSION TITLE
1.33.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#18819] [General]
# Notes
- For Home Assistant users: due to https://github.com/Koenkk/zigbee2mqtt/issues/18445 at least Home Assistant 2023.8.0 is required.

# Improvements
- [#6108](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6108) Expose `store_1` action for Müller Licht 404002 (@obrain17)
- [#6103](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6103) Support battery for Namron 4512764 (@geirra)
- [#6092](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6092) Add `scene_rename` command (@vanviegen)
- [#6066](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6066) Add label to exposes (@Drafteed)
- [#6078](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6078) Support fan mode, keypad lockout and OTA and remove pi heating demand for Zen Zen-01-W (@Codel1417)
- [#6067](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6067) Support schedule for TuYa TS0601_thermostat_3 and add `_TZE200_p3dbf6qs` and `_TZE200_rxntag7i` (@dyx052612)
- [#6079](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6079) Support tamper for TuYa TS0207_water_leak_detector (@oscrey)
- [#6070](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6070) Refactor and add totalApparentPower for 3 phases data for LiXee (@arenoux)
- [#6052](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6052) Support OTA for BTicino L4411C/N4411C/NT4411C and K4003C/L4003C/N4003C/NT4003C (@RyuzakiKK)
- [#6049](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6049) Add OTA support for Bticino 4027C (@RyuzakiKK)
- [#342](https://github.com/Koenkk/zigbee-OTA/pull/342) Support OTA for Gledopto GL-D-007P  (@BradleyFord)
- [#6032](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6032) Add `led_in_dark` fromZigbee converter for Legrand + Bticino devices (@FabianMangold)
- [#742](https://github.com/Koenkk/zigbee-herdsman/issues/742) Support coordinator check (@Koenkk)

# Fixes
- [#6122](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6122) Add some Legrand/BTicino whitelabel models (@xraver)
- [#18797](https://github.com/Koenkk/zigbee2mqtt/issues/18797) Fix Iris 3460-L configure failing (@Koenkk)
- [#6119](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6119) Fix middle volume value for TuYa YXZBSL (@fliespl)
- [#6114](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6114) Enable hue/saturation for Paulmann lights (@sjorge)
- [#2191](https://github.com/Koenkk/zigbee2mqtt.io/pull/2191) Add ZWT-100-16A as whitelabel of X5H-GB-B (@Koenkk)
- [#18743](https://github.com/Koenkk/zigbee2mqtt/issues/18743) Disable unsupported `power_on_behaviour` for Aurora Lighting AU-A1ZBMPRO1ZX (@Koenkk)
- [#18693](https://github.com/Koenkk/zigbee2mqtt/issues/18693) Disable unsupported `power_on_behaviour` for Vimar 03981 (@Koenkk)
- [#6102](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6102) Override labels in exposes (@Drafteed)
- [#18656](https://github.com/Koenkk/zigbee2mqtt/issues/18656) Fix `Cannot read properties of undefined (reading 'hasOwnProperty')}` error for some Schneider Electric devices (@Koenkk)
- [#18651](https://github.com/Koenkk/zigbee2mqtt/discussions/18651) Fix OTA upgrade not working for Xiaomi ZNCLBL01LM (@Koenkk)
- [#18636](https://github.com/Koenkk/zigbee2mqtt/issues/18636) Fix Philips 9290022166 not controllable (@Koenkk)
- [#6068](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6068) Fix no firmware versions shown for Xiaomi RTCZCGQ11LM (@Nixon506E)
- [#18597](https://github.com/Koenkk/zigbee2mqtt/issues/18597) Fix Xiaoimi JTQJ-BF-01LM/BW device type and power source (@Koenkk)
- [#6071](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6071) Add missing actions to Philips 324131092621 (@Burki24)
- [#5864](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5864) Fix `_TZ3000_lepzuhto` not detected as supported (@Koenkk)
- [#17749](https://github.com/Koenkk/zigbee2mqtt/issues/17749) Fix Niko 552-721X2 state incorrect when controlled physically (@Koenkk)
- [#18558](https://github.com/Koenkk/zigbee2mqtt/issues/18558) Remove unsupported `power_outage_count` from Xiaomi GZCGQ01LM (@Koenkk)
- [#6062](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6062) Remove unsupported `tamper` from TuYa ZM-35H-Q, TS0202 and IH012-RT01 (@Chartreusito)
- [#6063](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6063) Update description of TuYa TS0207_water_leak_detector_2 (@Chartreusito)
- [#6057](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6057) Remove unsupported color for ADEO IA-CDZFB2AA007NA-MZN-02 and IG-CDZB2AG009RA-MZN-01 (@matco)
- [#18540](https://github.com/Koenkk/zigbee2mqtt/issues/18540) Fix Eco-Dim.05 not detected as supported (@Koenkk)
- [#18531](https://github.com/Koenkk/zigbee2mqtt/issues/18531) Fix OTA endpoint missing ofr Xiaomi GZCGQ11LM (@Koenkk)
- [#6042](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6042) Fixes for Third Reality 3RSNL02043Z (@llhappier)
- [#6038](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6038) Fix Lixee tempo tarf & review RP state (@SilentT-FR)
- [#18513](https://github.com/Koenkk/zigbee2mqtt/issues/18513) Fix TuYa TS0201 `_TZ3210_ncw88jfq` humidity off by a factor of 10 (@Koenkk)
- [#6033](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6033) Various fixes for ptvo.switch (@ptvoinfo)
- [#18718](https://github.com/Koenkk/zigbee2mqtt/issues/18718) Fix adding backup missing devices to backup (@Koenkk)
- [#749](https://github.com/Koenkk/zigbee-herdsman/issues/749) Emit disconnect event when Zigate unplugged (@yateeshc)
- [#18706](https://github.com/Koenkk/zigbee2mqtt/discussions/18706) Fix backup devices missing logging (@Koenkk)
- [#746](https://github.com/Koenkk/zigbee-herdsman/issues/746) Don't remove devices with linkkey from backup if they are still present in the database (@Koenkk)
- [#744](https://github.com/Koenkk/zigbee-herdsman/issues/744) Fix some commands failing due to policy. https://github.com/Koenkk/zigbee2mqtt/issues/18382 (@Koenkk)

# New supported devices
This release adds support for 41 devices: 
- [#18772](https://github.com/Koenkk/zigbee2mqtt/issues/18772) `QBKG32LM` Xiaomi Aqara smart wall switch H1 Pro (with neutral, tripple rocker) (@ssnaveen10)
- [#6121](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6121) `L101Z-SBI` Lytko Single channel Zigbee thermostat (@kirovilya)
- [#6121](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6121) `L101Z-SBN` Lytko Single channel Zigbee thermostat (@kirovilya)
- [#6121](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6121) `L101Z-SLN` Lytko Single channel Zigbee thermostat without screen (@kirovilya)
- [#6121](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6121) `L101Z-DBI` Lytko Dual channel Zigbee thermostat (@kirovilya)
- [#6121](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6121) `L101Z-DBN` Lytko Dual channel zigbee thermostat (@kirovilya)
- [#6121](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6121) `L101Z-DLN` Lytko Dual channel Zigbee thermostat without screen (@kirovilya)
- [#6120](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6120) `RBSH-RTH0` Bosch Room thermostat II (@miczab)
- [#18745](https://github.com/Koenkk/zigbee2mqtt/issues/18745) `TS030F` TuYa Smart blind controller (@JuMi2006)
- [#6115](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6115) `929003123801` Philips Hue white ambiance 6" downlight (@suspended)
- [#6116](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6116) `PJ-1203A` TuYa Bidirectional energy meter with 80A current clamp (@dams74)
- [#18326](https://github.com/Koenkk/zigbee2mqtt/issues/18326) `07752L` Immax NEO smart internal double socket (@Koenkk)
- [#6106](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6106) `929003054401` Philips Hue white ambiance suspension Fair (@markkvdb)
- [#18732](https://github.com/Koenkk/zigbee2mqtt/issues/18732) `4512760` Namron Zigbee dimmer 400W (@erwahlb)
- [#18712](https://github.com/Koenkk/zigbee2mqtt/discussions/18712) `5047330P6` Philips Hue white and color ambience Argenta spot black (3 spots) (@Koenkk)
- [#6104](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6104) `929003046601` Philips Philips Hue white ambiance pillar double spot (2 spot) with Bluetooth (@wterpstra)
- [#6100](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6100) `13190230` Philips Hue white dimmer 1-10V (@posixx)
- [#6101](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6101) `4512764` Namron Zigbee water leak sensor (@einarjh)
- [#6098](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6098) `HA-ZBM-MW2` Halemeier S-Mitter basic MultiWhite² 1-channel sender Zigbee  (@guerda)
- [#6093](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6093) `10454471` HORNBACH FLAIR Viyu Smart LED bulb RGBW E27 (@stnokott)
- [#6090](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6090) `600087L` Legrand Wireless and batteryless blind control switch (@Gautomax)
- [#6088](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6088) `SBTZB-110` Develco Smart button (@bitbytie)
- [#6085](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6085) `9290030518` Philips Philips filament E26 bulb (@kattmang)
- [#6084](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6084) `ZG-205Z/A` TuYa 24Ghz Human presence sensor (@13717033460)
- [#6080](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6080) `915005988001` Philips Hue Gradient light tube compact black (@mpaasch)
- [#18595](https://github.com/Koenkk/zigbee2mqtt/issues/18595) `MEG5126-0300` Schneider Electric Merten MEG5165 PlusLink relais insert with Merten Wiser System M push button (2fold) (@tech2mar)
- [#6069](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6069) `LM4110ZB` Sinopé Tank level monitor (@Nixon506E)
- [#6053](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6053) `HA-ZSM-MW2` Halemeier S-Mitter MultiWhite2 smart remote control (@yGuy)
- [#6053](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6053) `E0040006` Paul Neuhaus Q RGBW remote controller (@yGuy)
- [#17871](https://github.com/Koenkk/zigbee2mqtt/issues/17871) `LP_CF_7904008_EU` Philips MasterConnect LEDtube EM/mains T8 (@Koenkk)
- [#6048](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6048) `SZ-WTD03` Sercomm Water leak detector (@gregwjacobs)
- [#18237](https://github.com/Koenkk/zigbee2mqtt/issues/18237) `ZY-M100-24G` TuYa 24G MmWave radar human presence motion sensor (@VladKorr)
- [#17377](https://github.com/Koenkk/zigbee2mqtt/issues/17377) `CCTFR6100Z3` Schneider Electric Wiser radiator thermostat (@Koenkk)
- [#6045](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6045) `MTG075-ZB-RL` TuYa 5.8G human presence sensor with relay (@MityaDVG)
- [#6046](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6046) `RS 232 C` Innr GU10 spot, dimmable, RGBW (@Koenkk)
- [#6035](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6035) `84845506` ADEO ENKI LEXMAN Gdansk (@hkienlen)
- [#6041](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6041) `IH012-RT02` TuYa Motion sensor (@timiti29)
- [#18465](https://github.com/Koenkk/zigbee2mqtt/issues/18465) `8719514419278` Philips Hue Ellipse E27 smart bulb (@Koenkk)
- [#18471](https://github.com/Koenkk/zigbee2mqtt/issues/18471) `LGYCDD01LM` Xiaomi Aqara Zigbee 3.0 LED strip T1 (@FabienVINCENT)
- [#6037](https://github.com/Koenkk/zigbee2mqtt/issues/6037) `CGW-Z-0010` CEL Cortet range extender (@baudfather)
- [#6036](https://github.com/Koenkk/zigbee2mqtt/issues/6036) `ZG-205ZL` TuYa 24Ghz human presence sensor (@makstech)

# Fixed device detections
- [#18803](https://github.com/Koenkk/zigbee2mqtt/issues/18803) Detect `_TZ3210_weaqkhab` as Lonsonho QS-Zigbee-D02-TRIAC-L_1 (@Koenkk)
- [#6117](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6117) Detect `_TZ3000_aa5t61rh` as TS0002_switch_module_3 (@marcinsu)
- [#6111](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6111) Detect `_TZE204_rhblgy0z` as TuYa TS0601_din_1 (@alray31)
- [#6112](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6112) Detect `929002376402` as 929002376401 (@DrHauss)
- [#6099](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6099) Detect `_TYZB01_2jzbhombz` as SBDV-00029, `_TYZB01_ub7urdza` as SBDV-00032 and `_TYZB01_epni2jgy` as SBDV-00030 (@kinjalik)
- [#18634](https://github.com/Koenkk/zigbee2mqtt/discussions/18634) Detect `HK_DIM_A` as `HK-DIM-A` (@Koenkk)
- [#6096](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6096) Detect `NimlyTouch` as Onesti Products AS easyCodeTouch_v1 (@DewGew)
- [#18612](https://github.com/Koenkk/zigbee2mqtt/issues/18612) Detect all `TS0225` as supported (@Koenkk)
- [#17704](https://github.com/Koenkk/zigbee2mqtt/issues/17704) Detect `_TZ3000_mh9px7cq` as TuYa TS0044_1 (@Koenkk)
- [#6081](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6081) Detect `_TZ3000_qystbcjg` as UNSH SMKG-1KNL-EU-Z (@Vorta)
- [#18633](https://github.com/Koenkk/zigbee2mqtt/issues/18633) Detect `_TZE204_nklqjk62` as TuYa PJ-ZGD01 (@Koenkk)
- [#6076](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6076) Detect `_TZE200_ves1ycwx` as TuYa SPM02 (@script0803)
- [#6065](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6065) Detect `_TZ3000_msl6wxk9` as TuYa ZMS-102 (@Chartreusito)
- [#18569](https://github.com/Koenkk/zigbee2mqtt/issues/18569) Detect `Remote Control`/`MLI` as Müller Licht 404049D (@Koenkk)
- [#5930](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5930) Detect `_TZE204_dtzziy1e` as TuYa MTG075-ZB-RL (@Koenkk)
- [#6058](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6058) Detect `RGBW` as Paulmann 371000002 (@arteck)
- [#10397](https://github.com/Koenkk/zigbee2mqtt/issues/10397) Detect `_TYZB01_j7iyqfcs` as TuYa TS0202 (@Koenkk)
- [#6056](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6056) Detect `_TZ3000_odzoiovu` as TuYa TS0003_switch_module_2 (@MityaDVG)
- [#6044](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6044) Detect `_TZ1800_ho6i0zk9` as Lidl HG06336 (@chazsconi)
- [#18516](https://github.com/Koenkk/zigbee2mqtt/issues/18516) Detect `_TZ3210_x13bu7za` as Lidl 399629_2110  (@Koenkk)
- [#6043](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6043) Detect `NimlyIn` as Onesti Products AS easyCodeTouch_v1 (@Koenkk)
- [#18522](https://github.com/Koenkk/zigbee2mqtt/issues/18522) Detect `PSMP5_00.00.03.05TC` as Lupus 12050 (@Koenkk)
- [#18512](https://github.com/Koenkk/zigbee2mqtt/issues/18512) Detect `_TZ3210_cieijuw1` as Nous P3Z (@Koenkk)
- [#18494](https://github.com/Koenkk/zigbee2mqtt/issues/18494) Detect `_TZ3210_j4pdtz9v` as TuYa TS0001_fingerbot (@Koenkk)

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.33.0'>1.33.0</a>.</em>

**Comment by rhvs** [comment]
Could you please point towards the location of the coordinator check? https://github.com/Koenkk/zigbee2mqtt/issues/18426

**Reply by Koenkk** [reply]
https://www.zigbee2mqtt.io/guide/usage/mqtt_topics_and_messages.html#zigbee2mqtt-bridge-request-coordinator-check

**Comment by xconverge** [comment]
@Koenkk  The "latest" docker tag is 1.32.0 not 1.33.0, just a headsup

**Reply by BradleyFord** [reply]
just noticed the same

**Reply by evoteknic** [reply]
Yes, it has happened to me too, after updating the container, the version that appears is not 1.33, it is 1.32.0 commit: 69638d7

**Reply by Koenkk** [reply]
Thanks for letting me know, should be ok now!

**Comment by OrionExplorer** [comment]
Hi @Koenkk, could you please deploy new zigbee2mqtt.io?

**Reply by Koenkk** [reply]
Whoops, done!

**Comment by tititg34** [comment]
Hello, this new version break the support for Philips Hue Gradient large black EU light tube (915005988201)

**Comment by salopette** [comment]
hi, with the last update, the front end is always displayed as night mode, if I switch it to light, it comes back to dark after visiting the site again

**Reply by keithr59** [reply]
I experience the same using the Opera web browser but it displays correctly as light on both Microsoft Edge and the Android companion app,

**Reply by kwetnico** [reply]
Yes me too, on Google chrome

**Reply by Nardol** [reply]
Same with FireFox 117.0 (Debian).
I think this issue should be reported to https://github.com/nurikk/zigbee2mqtt-frontend

**Reply by burpy** [reply]
Same in Safari 16.5.2

**Reply by xraver** [reply]
Same here google chrome and edge

**Comment by kiekerjan18** [comment]
> https://github.com/Koenkk/zigbee2mqtt/discussions/18651 Fix OTA upgrade not working for Xiaomi ZNCLBL01LM (@Koenkk)

Fix did not work here. 

![image](https://github.com/Koenkk/zigbee2mqtt/assets/63964079/fff25980-c8a7-4cbd-8e1f-12dff8f816c3)

**Comment by teleoltre** [comment]
Hi, 
My Sonoff Zigbee version E dongle with router firmware is no longer detected by the system after the upgrade.

I also tried reinstalling the firmware but nothing worked...

it's normal ?

**Comment by burpy** [comment]
I can not create, delete or update a group (over the GUI) and one of my groups with 7 devices and 2 scenes is now empty. Downgrade to 1.32.0 and it worked again. I had to add the devices again but it works with 1.32.0

**Comment by henkiejan1** [comment]
I don't want to be impatient but are there some troubles with publishing 1.34.0? For a very longtime every first of the new month a new release was published but there isn't some new release?

**Reply by Koenkk** [reply]
I was on a trip, release will come today.

**Reply by henkiejan1** [reply]
You deserve your time! Hope you has a great trip. And i don't want be impatient. But there are some things for sure in life and that's there is a sunrise and sunset everyday and there will be a Zigbee2MQTT release on the first of every month. :stuck_out_tongue_winking_eye: But thank you for all your great work!
